sql/step_a111_settings.sql

```diff
@@ -0,0 +1,95 @@
+-- Drop the existing table
+DROP TABLE IF EXISTS OH_SETTINGS;
+
+-- Create settings table structure
+CREATE TABLE OH_SETTINGS (
+  SETT_ID INT(11) NOT NULL AUTO_INCREMENT COMMENT 'Default generated ID',
+  SETT_CODE VARCHAR(50) NOT NULL,
+  SETT_VALUE_TYPE ENUM('bool', 'number', 'text', 'select') NOT NULL DEFAULT 'text',
+  SETT_VALUE_OPTIONS VARCHAR(500) NULL DEFAULT NULL COMMENT "Comma-separated list of possible values, in case of type 'select'",
+  SETT_DEFAULT_VALUE VARCHAR(255) NOT NULL,
+  SETT_VALUE VARCHAR(255) NOT NULL,
+  SETT_DESCRIPTION VARCHAR(500) NULL,
+  SETT_CREATED_BY VARCHAR(50) NULL DEFAULT NULL,
+  SETT_MODIFIED_BY VARCHAR(50) NULL DEFAULT NULL,
+  SETT_CREATED_DATE DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP(),
+  SETT_MODIFIED_DATE DATETIME NULL DEFAULT NULL,
```

Comment:
normally the fields and their order are the following:
```suggestion
  SETT_CREATED_BY VARCHAR(50) NULL DEFAULT NULL,
  SETT_CREATED_DATE DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP(),
  SETT_LAST_MODIFIED_BY VARCHAR(50) NULL DEFAULT NULL,
  SETT_LAST_MODIFIED_DATE DATETIME NULL DEFAULT NULL,
```